cmd/olfullnode/devnet.go

```diff
@@ -667,7 +668,14 @@ func initialState(args *testnetConfig, nodeList []node, option ethchain.ChainDri
 			Name:             node.validator.Name,
 			Amount:           *balance.NewAmountFromInt(node.validator.Power),
 		}
+		wt := identity.Witness{
+			Address:     node.validator.Address.Bytes(),
+			PubKey:      pubkey,
```

Comment:
Does a witness need this ? or just the ECDSA key is enough ?

Reply:
I'm not sure if the other key is really used, I just wanted to do the refactor to use an actual witness struct. The implementation of the Witness itself is something I didn't want to modify.

Reply:
makes sense .